citrination/commands/create_dataset.py

```diff
@@ -17,9 +19,12 @@ def main(self):
     def setup(self):
         """Setup this application."""
         cli.app.CommandLineApp.setup(self)
-        self.add_param("-k", "--api_key", help="Citrination API key.", required=True)
-        self.add_param("-e", "--citrination_host", help="Citrination host.",
-                       default='https://citrination.com', required=False)
+        self.add_param("-p", "--project", default=None, required=False,
```

Comment:
This change makes it impossible to run the cli against a non-citrination.com domain (like local server)